### PR TITLE
feat: inspector: fbx/animator  add import_all_animator config

### PR DESCRIPTION
**Pull request by dogodo-cc**
Re: https://github.com/cocos/3d-tasks/issues/15362

### Changelog

* inspector: fbx/animator  add import_all_animator config

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/assets/fbx/animation.js [hunk 1]
```diff
@@ -659,6 +672,31 @@ const Elements = {
             Object.assign(panel.$.controlRight.style, panel.currentClipInfo.ctrlEndStyle);
         },
     },
+    importAllAnimator: {
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/animation.js:675]: follow the prop name 'mountAllAnimationsOnPrefab'

Comment (arsen2010) [inline editor/inspector/assets/fbx/animation.js]: follow the prop name 'mountAllAnimationsOnPrefab'

File: editor/inspector/assets/fbx/animation.js [hunk 2]
```diff
@@ -313,6 +324,8 @@ ui-section {
 
 exports.$ = {
     container: '.container',
+    importAllAnimator: '.animator-config-import-all-animator ui-checkbox',
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/animation.js]: suggest the name use 'importAllAnimationsCheckbox'

**Review by arsen2010 — APPROVED** [review]

File: editor/i18n/en/assets.js [hunk 3]
```diff
@@ -285,6 +285,9 @@ module.exports = {
                     name: 'Skip Validation',
                     title: 'Skip validation of the model file.',
                 },
+                mountAllAnimationsOnPrefab: {
+                    name: 'Import all animations in the prefab',
```
Comment (shrinktofit) [inline editor/i18n/en/assets.js]:
```suggestion
                    name: 'Mount All Animations Onto Prefab',
```

File: editor/i18n/zh/assets.js [hunk 4]
```diff
@@ -285,6 +285,9 @@ module.exports = {
                     name: '跳过验证',
                     title: '跳过对模型文件的验证。',
                 },
+                mountAllAnimationsOnPrefab: {
+                    name: '挂载全部动画',
```
Comment (shrinktofit) [inline editor/i18n/zh/assets.js]:
```suggestion
                    name: '挂载全部动画到预制体',
```

File: editor/i18n/en/assets.js (hunk 3 above)
Comment (shrinktofit) [reply editor/i18n/en/assets.js]: @AndyYuTian Please review the option display name

File: editor/i18n/zh/assets.js (hunk 4 above)
Comment (shrinktofit) [reply editor/i18n/zh/assets.js]: @AndyYuTian Please review the option display name

**Review by arsen2010 — APPROVED** [review]

File: editor/inspector/assets/fbx/animation.js [hunk 5]
```diff
@@ -1115,6 +1153,12 @@ exports.methods = {
         panel.dispatch('change');
         panel.dispatch('snapshot');
     },
+    setProp(prop, type, event) {
+        setPropValue.call(this, prop, type, event);
+
+        this.dispatch('change');
+        this.dispatch('track', { tab: 'model', prop, value: event.target.value });
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/animation.js]: track tab is not model , maybe 'animation'

**Review by arsen2010 — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**